<launch>
    <node name="motorCommander" pkg="jaguarbot" type="motorCommander.py"/>
    <node name="triggerer" pkg="jaguarbot" type="triggererSim.py"/>
</launch>